src/test/java/com/rackspace/ceres/app/services/MetadataServiceTest.java

```diff
@@ -93,20 +139,18 @@ void storeMetadata() {
     store(tenantId, metricName, "linux", "h-3", "dev");
     store(tenantId, metricName, "linux", "h-3", "dev");
 
+
     assertThat(cassandraTemplate.select(Query.query(), SeriesSet.class).collectList().block())
         .containsExactlyInAnyOrder(
-            seriesSet(
-                tenantId, metricName, "deployment", "dev", "deployment=dev,host=h-3,os=linux"),
-            seriesSet(
-                tenantId, metricName, "deployment", "prod", "deployment=prod,host=h-1,os=linux"),
-            seriesSet(
-                tenantId, metricName, "deployment", "prod", "deployment=prod,host=h-2,os=windows"),
-            seriesSet(tenantId, metricName, "host", "h-1", "deployment=prod,host=h-1,os=linux"),
-            seriesSet(tenantId, metricName, "host", "h-2", "deployment=prod,host=h-2,os=windows"),
-            seriesSet(tenantId, metricName, "host", "h-3", "deployment=dev,host=h-3,os=linux"),
-            seriesSet(tenantId, metricName, "os", "linux", "deployment=dev,host=h-3,os=linux"),
-            seriesSet(tenantId, metricName, "os", "linux", "deployment=prod,host=h-1,os=linux"),
-            seriesSet(tenantId, metricName, "os", "windows", "deployment=prod,host=h-2,os=windows")
+            seriesSet(tenantId, metricName, "deployment", "dev", seriesSet3),
```

Comment:
Nit: It confused me that these seriesSet's, like seriesSet3, are supposed to be hashes but they are generated by the unhashedSeriesSet() method.  A clarifying comment would reduce the confusion.

Reply:
Good point, will do